content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,78 @@
+
+Personally, I did not like the style of the presentation (not the theme of slides, but how they got presented! :D) but the information provided is very important and so you should take a look here: https://derickrethans.nl/talks/php-phpday20 
+
+#Symfony 5, the new bit 
+## Andreas Hucks ([@meandmymonkey](https://twitter.com/meandmymonkey))
+
+A deep dive into Symfony 5 features and enhancements. Knowing where the framework is heading will make us take better-informed decisions in our day to day work.
+
+#Leveraging Typed Exceptions for Cleaner Error Handling 
+## Chris Holland ([@chrisholland](https://twitter.com/chrisholland))
+
+A talk suited mostly to entry-level/mid-level developers, but indeed useful to learn (or recap) how to establish a lifelong relationship with exceptions: they are not enemies, they can help us contextualizing the scope in which they occur and so speed up the debugging process.
+Slides available: https://bit.ly/exceptions-ftw
+ 
+#Kubernetes for PHP developers 
+## Alessandro Lai ([@AlessandroLai](https://twitter.com/AlessandroLai))
+
+A well-presented talk by my colleague Alessandro - Too much dashboard screenshot for me, but that's only because I am more dev than ops ;)
+
+#Technically Speaking: Improve your code with documentation 
+## Alexandra White ([@heyawhite](https://twitter.com/heyawhite))
+
+Brilliantly presented by Alexandra! A tidbit that I brought home: there are moments to be (like) Shakespeare and others not. One of those when not to be Shakespeare is when writing documentation.
+I must say that this talk lighted my interest in technical writing.
+The cherry on the cake was that the presentation had subtitles, which should definitely become a de-facto standard at least for pre-recorded videos. Very inclusive!
+Slides: https://docs.google.com/presentation/d/1l7cjrouAGdsGae4nXyJLcGcool1l0zx5knXvL_4h04Q/edit#slide=id.g589a77319a_0_34
+
+#Looping the Loop with SPL Iterators 
+## Mark Baker ([@Mark_Baker](https://twitter.com/Mark_Baker))
+
+A tour of the not-well-known iterators world and a little mea-culpa about the initially almost-not-existent SPL documentation! 🙂 
+Slides: https://www.slideshare.net/MarkBakerUK/looping-the-loop-with-spl-iterators/
+
+#Getting started with ReactPHP - Pushing real-time data to the browser 
+## Christian Lück ([@another_clue](https://twitter.com/another_clue))
+
+This is the real React! Not that other little thingy made by Facebook! An introductive talk to the react world and the world is based on: async and event-driven programming and of course promises (https://music.youtube.com/watch?v=Ok1Z65OMi1Y&feature=gws_kp_track), 
+
+#Pragmatic Optimism: How PHP Conquered the Web
+## Samantha Quiñones ([@ieatkillerbees](https://twitter.com/ieatkillerbees))
+
+I really enjoyed the final talk that was a historic retrospective of the web development, starting from Charles Babbage up to (fast forward) CGI, the Personal Home Page Tools - PHP Tools and its evolution.
```

Comment:
historical retrospective on the development of the Web,

Reply:
it was not on the development of web, but web-depelopment (aka programming)

Reply:
Changed in 
I really enjoyed the final talk that was a historical retrospective of the web and web developing,